frontend/public/components/resource-quota.jsx

```diff
@@ -298,9 +298,9 @@ const Details = ({ obj: rq }) => {
               )}
             </p>
             <p>
-              {`${t(
-                'resource-quota~A cluster administrator can establish limits on both the amount you can Request and your Limits with a',
-              )} ${ResourceQuotaModel.label}${t('resource-quota~.')}`}
+              {t(
+                'resource-quota~A cluster administrator can establish limits on both the amount you can request and your Limits with a ResourceQuota.',
```

Comment:
I’d make limits lowercase here too since it’s not a k8s resource

Reply:
All set!